.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "go.testFlags": ["-v"]
+}
```

Comment:
please do not include this file in the pr
I see the repo doesn't have a .gitignore file - maybe even include one that ignores visual studio code and golang artifacts? (just a suggestion)